server/client.ts

```diff
@@ -337,6 +337,7 @@ class Client {
 		// TODO
 		// eslint-disable-next-line @typescript-eslint/no-misused-promises
 		events.forEach(async (plugin) => {
+			// @ts-ignore
```

Comment:
> Dynamic imports are only supported when the '--module' flag is set to 'es2020', 'es2022', 'esnext', 'commonjs', 'amd', 'system', 'umd', 'node16', or 'nodenext'.

Why is this safe to ignore?
At the very least it needs a comment as to why that's ok :wink: better yet, solving the actual root problem I guess.

Reply:
The client is being built for `es2015`. I didn't really want to drop support for old clients especially as these dynamic imports only happen in the server side app, and are just included due to transitive dependencies. 

Let me see what I can make happen with moving around some code.